crates/interpreter/src/instructions/host.rs

```diff
@@ -72,6 +73,13 @@ pub fn extcodehash<H: Host + ?Sized, SPEC: Spec>(interpreter: &mut Interpreter,
     } else {
         gas!(interpreter, 400);
     }
+    if let Some((bytecode, _)) = host.code(address) {
```

Comment:
This needs to go where `Host` is implemented here: https://github.com/bluealloy/revm/blob/561fbdbb117a5613475a8f099981a36fedd0104b/crates/revm/src/context.rs#L156

host.code_hash needs to return whatever is needed for instruction.

Reply:
makes sense, let me do it 💪